DRAFT: CDK Overview update (#2115)
0xPolygon/polygon-docs: docs/cdk/overview.md

comments: true
---

The Polygon Chain Development Kit (CDK) is a modular, open-source software toolkit that enables blockchain developers to configure ZK-powered chain architectures.

With Polygon CDK, developers can launch new chains running the Polygon zkEVM protocol as Layer 2s on Ethereum.

Developers can configure ZK-powered rollups, validium networks, or even sovereign chains with their own tailored finality mechanisms.

## Trustless finality

Why does Polygon CDK focus on ZK-powered chains?

> The power of zero-knowledge technology is in enabling chains to achieve trustless finality, where chain users do not have to rely on a few individuals to confirm the finality of their transactions.

Author: I avoided a super long sentence but yours flows.

With Polygon CDK chains developers can choose between a rollup and validium chain, and decide on how they want to configure the chain's proving system.

> Polygon launches the CDK rollup/validium mode with which developers can configure CDK components that run the Polygon zkEVM protocol.

Reviewer: ```suggestion
Polygon has launched the CDK rollup/validium mode enabling developers to configure CDK components that run the Polygon zkEVM protocol.
```

> ## Polygon CDK benefits

Reviewer: ```suggestion
## Key advantages
```

> Polygon CDK provides components necessary to build a layer 2 blockchain that is secure, scalable, and interoperable with other chains.

Reviewer: ```suggestion
Polygon CDK provides the essential components to build a Layer 2 blockchain that is secure, scalable, and interoperable with other chains.
```

- Security: CDK builds highly secure, scalable L2s that utilize the latest innovations in zero-knowledge technology.
- High-performance: CDK Erigon implements fast-syncing Erigon clients that run the battle-tested Polygon zkEVM protocol.
- Modularity: CDK modular components allow developers to easily customize their L2 environment and build a chain that meets their specific needs.
- Interoperability: Opt-in to the [AggLayer](../agglayer/overview.md) to bootstrap your chain’s ecosystem, enable cross-chain transactions while expanding your reach, user base, and liquidity from other established chains.
- Sovereignty: Maintain full control over your chain’s revenue, governance, security, economic policies, and more.

> - Low gas fees: Transaction fees are orders of magnitude lower than in Ethereum, and processed substantially faster. This enables a fast, cheap, and secure user experience unaffected by any high activity experienced on shared networks.

Reviewer: ```suggestion
- Low gas fees: Transaction fees are orders of magnitude lower than those on Ethereum and are processed substantially faster. This enables a fast, ultra low-cost, and secure user experience unaffected by any high activity experienced on shared networks.
```

> ### **Where to now?**

Reviewer: ```suggestion
### Next steps
```

> - [Deploy a local CDK on Kurtosis](../cdk/getting-started/local-deployment.md). Follow the guide to deploy a CDK stack on your local machine.

Reviewer: ```suggestion
- [Deploy a local CDK environment using Kurtosis](../cdk/getting-started/local-deployment.md). Follow the guide to deploy a CDK stack on your local machine.
```

> - Check out the [concepts documentation](../cdk/concepts/layer2s.md) to understand the CDK at a high level.

Reviewer: ```suggestion
- Check out the [concepts documentation](../cdk/concepts/layer2s.md) to gain a high-level understanding of the CDK.
```
